ablateLibrary/domain/fileMesh.cpp

```diff
@@ -1,30 +1,24 @@
 #include "fileMesh.hpp"
 #include <utilities/petscError.hpp>
 #include <utilities/petscOptions.hpp>
-ablate::domain::FileMesh::FileMesh(std::string nameIn, std::filesystem::path pathIn, std::shared_ptr<parameters::Parameters> options, std::vector<std::shared_ptr<modifier::Modifier>> modifiers)
```

Comment:
Note that you can do this using `-dm_plex_filename`, so it is unclear whether we need this class, but maybe due to how the YAML is processed.

Reply:
Thanks!  It is not really needed, it must simplifies the setup in yaml so yaml is self explaining for the examples.